### ISSUE TITLE
TypeError: Cannot read property 'modes' of undefined

### ISSUE THREAD

**Issue by OR13** [digitalbazaar/did-method-key-js#23]
Now using node v12.16.0 (npm v6.14.4)

```
did.actor@1.0.0 /Users/orie/Desktop/Code/Transmute/did.actor
└── did-method-key@0.6.1 
```

```
const didKeyDriver = require("did-method-key").driver();
```

```
 FAIL  __tests__/make-degree.spec.js
  ● Test suite failed to run

    /Users/orie/Desktop/Code/Transmute/did.actor/node_modules/x25519-key-pair/node_modules/node-forge/lib/aes.js:1
    TypeError: Cannot read property 'modes' of undefined
    at Object.<anonymous> (node_modules/x25519-key-pair/node_modules/node-forge/lib/aes.js:254:43)
```

**Comment by dmitrizagidulin** [comment]
Thanks @OR13.

**Comment by OR13** [comment]
downgrading to `did-method-key@0.5.0` solved this for me.

**Comment by dmitrizagidulin** [comment]
Should be fixed in latest (v2.0), closing.
